Shared/Model/Activity.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Activity.swift
+//  Fun Gen
+//
+//  Created by Julian Burrington on 4/23/22.
+//
+
+import Foundation
+
+struct Activity: Codable {
+    var id: String
+    var title: String
+    var category: Category
+    var author: User
+    var members: [User] = []
```

Comment:
As discussed before, shared values should be "referenced" using ID instead of copying/duplicating the value. So here, the element type should be User.ID (which is `String`) instead of User

Reply:
If you really want to make it clear that the String should be User.ID, you can define a typealias within User like this:

```swift
struct User {
    typealias ID = String
    ...
}
```

then you can use User.ID

Reply:
Nice, I like the User.ID with typealias

Reply:
Gives warning:

> Type Name Violation: Type name should be between 3 and 40 characters long: 'ID' (type_name)

So I decided to add comments like the one below for all of the data types and their properties

```swift
/// Array of *Activity.id*
var activities: [String] = []
```

Reply:
Interesting. I think this is a case where we should slightly tweak SwiftLint rules, let me take care of that